docs/src/main/sphinx/functions/window.rst

```diff
@@ -14,7 +14,7 @@ For example, the following query ranks orders for each clerk by price::
     FROM orders
     ORDER BY clerk, rnk
 
-The window can be specified in two ways (see ``WINDOW Clause``):
+The window can be specified in two ways (see :ref:`window_clause`):
```

Comment:
it's weird to introduce an enumeration and _also_ link somewhere else as the source of information

i think the ref should be part of the first bullet in the list below (or both).

or maybe, this should be sth like "see there for more info" (then it would go below the list)

Reply:
I'm not changing those docs here. I just saw there was a missing reference when I was trying to link to that doc in the release notes.

Reply:
yes, i know; but you can still improve if you want